servlet/src/main/java/org/togglz/servlet/activation/ClientIpActivationStrategy.java

```diff
@@ -49,7 +49,10 @@ public boolean isActive(FeatureState featureState, FeatureUser user)
          List<String> parts = Strings.splitAndTrim(featureState.getParameter(PARAM_IPS), "[\\s,]+");
 
          try {
-            String remoteAddr = request.getRemoteAddr();
+             String remoteAddr = request.getHeader("X-FORWARDED-FOR");
+             if(Strings.isBlank(remoteAddr)) {
+                 remoteAddr = request.getRemoteAddr();
+             }                                                   
```

Comment:
Weird, i thought I ran the formatting with the togglz rules, however, I ran it again and I get a lot of changes. Please give it a look